File: Assignment3/SumOfTwoNumbers.java
import java.util.Scanner;

public class SumOfTwoNumbers {
	
	public int sumOfTwoNumbers(int num1,int num2) {
		
		int sum = num1 + num2;
		
		return sum;
	}
	public static void main(String[] args) {		
		Scanner sc = new Scanner(System.in);
		SumOfTwoNumbers sn = new SumOfTwoNumbers();
		
		System.out.println("Enter first number.");
		int num1 = sc.nextInt();
		System.out.println("Enter first number.");
		int num2 = sc.nextInt();
		
		int sum = sn.sumOfTwoNumbers(num1,num2);
		
		System.out.println("The sum of "+num1+" and "+num2+ " is " + sum);
		
	}

}

File: CPP/Assignment/Labwork/Cppday2/src/SumOfTwoNumbers.cpp
/*
 * SumOfTwoNumbers.cpp
 *
 *  Created on: 25-Aug-2026
 *      Author: Nilesh
 */


#include<iostream>

using namespace std;

int sumOfTwoNumbers(int num1,int num2){

	return num1 + num2;
}

int main(){

	int num1;
	int num2;

	cout<< "Enter the first number: " << endl;
	cin >> num1;
	cout<< "Enter the second number: " << endl;
	cin >> num2;

	int result = sumOfTwoNumbers(num1,num2);

	cout<< "Sum of "<< num1 << " and " << num2 <<" is : " << result;

	return 0;


}
